extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -109,7 +109,7 @@
 
 					<? if ( !empty( $user['website'] ) ): ?>
 						<li class="website">
-							<a href="<?= $user['website'] ?>" rel="nofollow">
+							<a href="<?= Sanitizer::encodeAttribute( $user['website'] ) ?>" rel="nofollow">
```

Comment:
Is the user prevented from inserting `javascript:` links into their website in the user-attribute service code now?